src/PubSubClient.h

```diff
@@ -76,6 +76,9 @@
 #ifdef ESP8266
```

Comment:
Maybe:

`#if defined(ESP8266) or defined(ESP32)
`

to avoid dups

Reply:
I'm not sure that or will work, but:
#if defined(ESP8266) || defined(ESP32) 

is working

Reply:
https://github.com/knolleary/pubsubclient/pull/336

already there :)